FSI/Force.H

```diff
@@ -56,11 +56,18 @@ public:
         */
     );
 
+    //- Constructor
+    Force
```

Comment:
Why do we need an additional constructor without timeName? We do have `timeName` available when we do `Stress : Force`.

Reply:
Correct, my fault

Reply:
Wait, the previous constructor also creates the force field and defines the dataType. The new constructor does nothing and is just for the access we need. I would like to keep it.

Reply:
Then we could also make the 3-argument constructor call the 2-argument constructor, to avoid duplication. This is also a common pattern in the OpenFOAM source.

Reply:
Like this?

Reply:
I am not sure what you are refering to, could you add the code here? In `Force.C` I see that the 3-argument constructor does everything and the 2-argument constructor does nothing.

Reply:
But the 3 argument constructor calls the 2 argument constructor. The 2 argument constructor initializes the const variables. That is indeed not very much, but exactly what we want for the inheritance. Do I understand correctly?

Reply:
I guess I was looking at an outdated version of the file (clicking on the code here). This looks good: https://github.com/precice/openfoam-adapter/pull/125/commits/e48b212ddc9334a377c541b8b62cdd9661d53c82